src/AccessibilityInsights.Extensions.GitHub/IssueReporter.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using AccessibilityInsights.Extensions.AzureDevOps;
+using AccessibilityInsights.Extensions.Interfaces.IssueReporting;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace AccessibilityInsights.Extensions.GitHub
+{
+    public class IssueReporter : IIssueReporting
+    {
+        private IssueReporter _instance;
+        private ConfigurationModel _configurationControl;
+
+        private IssueReporter()
+        {
+            _configurationControl = new ConfigurationModel();
+        }
+
+        public IssueReporter getDeafualtInstance()
+        {
+            if (_instance == null)
+            {
+                _instance = new IssueReporter();
+            }
+            return _instance;
+        }
+
+        public string ServiceName => Properties.Resources.extentionName;
+
+        public Guid StableIdentifier => new Guid ("bbdf3582-d4a6-4b76-93ea-ef508d1fd4b8");
```

Comment:
>"bbdf3582-d4a6-4b76-93ea-ef508d1fd4b8" [](start = 50, length = 38)

how about pulling this as const and explain what it is? 
At least for me, I don't know what StableIdentifier is for.

Reply:
This is described in `AccessibilityInsights.Extensions.Interfaces.IssueReporting.IIssueReporting`. Do we need to describe it here as well?
`const` only applies to numbers, Boolean values, strings, or a null reference, not Guids:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/const

Reply:
See AccessibilityInsights.Extensions.GitHubAutoUpdate\AutoUpdate.cs line 58 for how I generally handle documenting implementations of an interface

---
In reply to: [267090971](https://github.com/Microsoft/accessibility-insights-windows/pull/184#discussion_r267090971) [](ancestors = 267090971)